alacritty/src/cli.rs

```diff
@@ -277,7 +280,7 @@ impl Options {
             }
         }
 
-        config.set_dynamic_title(config.dynamic_title() && config.window.title.is_none());
+        config.set_dynamic_title(config.dynamic_title() && config.window.title() == DEFAULT_NAME);
```

Comment:
Didn't we say the only reason why we need title to be an `Option` is because we set dynamic title depending on it?

If we change things here to match against `DEFAULT_NAME`, we don't need it as an option anymore and as a result we don't need to have setters and getters, right? Or did I forget about a different reason we did things with optionals?

Seems like it might result in a cleaner solution too, we don't need to support setting the title to `Alacritty` to disable dynamic title anyways, so it should be fine.

Reply:
Understood. I pushed a commit that changes it from `Option<String>` to just `String`.